README.md

```diff
@@ -59,6 +59,12 @@ sh
   cd my-folder
  git clone https://github.com/ernestmusong/Math-Magicians.git
 
+ ### Install dependencies:
+ Run: npm install
+
+ ### Start the dev server:
+ Run: npm start
+
```

Comment:
- [ ] Good job adding these sections 👍👍 I think it would be better if you use the same heading as given in the Microverse README template. i.e, Instead of **_Install dependencies:_** you can use **_Install_** and similarly, for **_Start the dev server:_** you can use **_Usage_**. What do you think about this? 💭⚒